### PR TITLE
Cxtb 466  Verify emergency services button in portal

### PR DESCRIPTION
**Pull request by jmontero-nts**
proof of succeed:
image

### CODE REVIEW

File: tests/cases/seniors_app/case_seniors_app_home.yaml [hunk 1]
```diff
@@ -81,27 +81,9 @@ TestSeniorAppCallPersonalCarePartnerOnHomePage:
       Role: desktopChrome
       Strategy: xpath
       Id: $PAGE.care_platform_call_portal.video_btn$
-    - Type: wait_for
```
[the remark this answers is not in the document]
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml]: I already fixed, this was a mistake.

File: tests/cases/seniors_app/case_seniors_app_home.yaml [hunk 2]
```diff
@@ -1128,4 +1128,355 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
...
+    # Verify that the "Call" button is disabled when the phone number input is empty.
+    - Type: wait_for_property
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.call_button$
+      Property: disabled
+      Value: "true"
+    - Type: set_env_var
+      Value: 61285038000
+      Var: soft_call_phone_number
```
[the remark this answers is not in the document]
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml:1441]: I agree

File: tests/cases/seniors_app/case_seniors_app_home.yaml [hunk 3]
```diff
@@ -1128,4 +1110,353 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
...
   # Go back to the iPad and fill in the Post Call Survey.
     - Type: case
-      Value: SeniorGreatPostSurveyCallHelper
\ No newline at end of file
+      Value: SeniorGreatPostSurveyCallHelper
+
+# CXTB-466: Verify "Emergency Services" button in portal
+TestSeniorVerifyEmergencyServicesButton:
+  Vars:
+    TIME: 5
```
[the remark this answers is not in the document]
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml]: Done

File: tests/cases/seniors_app/case_seniors_app_home.yaml [hunk 4]
```diff
@@ -1128,4 +1128,343 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
...
+    - LoginNeverAloneHelper
+    - SeniorCleanPrompts
+    - CarePartnerCleanCallQueueAndHangedCalls
+  Aftercases:
+    - TerminateApp
+  Roles:
+    - Role: localiOS
+      App: NeverAlone
+    - Role: desktopChrome
+      Capabilities:
```
Comment (dguzmanneural) [inline tests/cases/seniors_app/case_seniors_app_home.yaml:1148]: Capabilities is empty. Can you check if there's something missing, if not please delete it.

**Review by dguzmanneural — CHANGES_REQUESTED** [review]

File: tests/cases/seniors_app/case_seniors_app_home.yaml
Comment (csanchez-nts) [inline tests/cases/seniors_app/case_seniors_app_home.yaml:1]: Add test to SetSeniorNeverAloneAppTest

File: tests/cases/seniors_app/case_seniors_app_home.yaml [hunk 5]
```diff
@@ -1128,4 +1128,343 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
       Role: desktopChrome
   # Go back to the iPad and fill in the Post Call Survey.
     - Type: case
-      Value: SeniorGreatPostSurveyCallHelper
\ No newline at end of file
+      Value: SeniorGreatPostSurveyCallHelper
+
+# CXTB-466: Verify "Emergency Services" button in portal
```
Comment (csanchez-nts) [inline tests/cases/seniors_app/case_seniors_app_home.yaml:1133]: You need to create a manual test case to link to the automation. I couldn't find the test case for this ticket

File: tests/cases/seniors_app/case_seniors_app_home.yaml (hunk 4 above)
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml]: Done

File: tests/cases/seniors_app/case_seniors_app_home.yaml
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml:1]: Done

File: tests/cases/seniors_app/case_seniors_app_home.yaml (hunk 5 above)
Comment (jmontero-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml:1133]: Done and I saved it in test repository in never alone seniors app folden

Comment (csanchez-nts) [reply tests/cases/seniors_app/case_seniors_app_home.yaml:1133]: Need to add that test case number not the automation ticket number here

**Review by csanchez-nts — APPROVED** [review]

**Review by dguzmanneural — APPROVED** [review]

**Pull request merged**